src/Microsoft.CodeAnalysis.Analyzers/Core/CodeAnalysisDiagnosticsResources.resx

```diff
@@ -374,4 +374,13 @@
   <data name="EnableConcurrentExecutionFix" xml:space="preserve">
     <value>Enable concurrent execution</value>
   </data>
+  <data name="ClassIsNotDiagnosticAnalyzerDescription" xml:space="preserve">
+    <value>Types marked with DiagnosticAnalyzerAttribute(s) should inherit from DiagnosticAnalyzer.</value>
+  </data>
+  <data name="ClassIsNotDiagnosticAnalyzerMessage" xml:space="preserve">
+    <value>Inherit from DiagnosticAnalyzer or remove the DiagnosticAnalyzerAttribute(s).</value>
+  </data>
+  <data name="ClassIsNotDiagnosticAnalyzerTitle" xml:space="preserve">
+    <value>Class is not a DiagnosticAnalyzer.</value>
```

Comment:
I would probably just use `Types marked with DiagnosticAnalyzerAttribute(s) should inherit from DiagnosticAnalyzer.` as the Title and avoid having a description at all.